src/runner.rs

```diff
@@ -25,7 +25,48 @@ pub(crate) enum TickFlow {
     Exit,
 }
 
-pub(crate) const HELP_TEXT: &'static str = r"To start a transaction, enter 'start transaction' or 'begin'. To exit, enter 'exit' or press CTRL-D.";
+pub(crate) const HELP_TEXT: &'static str = r#"Shell Keys
+  Enter
+    - Runs the statement.
+  Escape+Enter (macOS)
+  Shift+Enter (Windows)
+    - Starts a new line to enter a statement that spans multiple lines. You can also copy input text with multiple lines and paste it into the shell.
+  Ctrl+C
+    - Cancels the current command.
+  Ctrl+D
+    - EOF / exit current level of shell. If not in a transaction, exit shell. If in a transaction, aborts the transaction.
+
+Database commands
+  start transaction
+  begin
+    - This starts a transaction.
```

Comment:
perhaps Mark can suggest some language here -- here's what i put in the original issue #168

  - This starts a transaction. Transactions currently have a timeout of 30 seconds. By default, auto-commit is enabled, and terminator is not required -- these are both configurable. To enter multi-line statements, use esc-enter. For more help with transactions go to https://docs.aws.amazon.com/qldb/latest/developerguide/data-shell.html#data-shell-transactions

Reply:
re-reading this now, it doesn't seem right - if they're using start transaction or begin, they're not in auto commit.

also, since we include the shell key reference we don't need to include multi-line details.

How about something like this? Probably need Mark's expertise here to confirm but I'm pulling this from the draft dev guide:

> start transaction
> begin
>   - Manually control transactions. You can run multiple statements within a transaction interactively, or non-interactively by batching commands and statements sequentially. Note: currently, QLDB limits transaction timeout to 30 seconds. If not committed within 30 seconds, QLDB automatically expires the transaction and rejects any changes made during the transaction.

Reply:
Makes sense and sounds pretty good to me. Just a few minor edits I'd suggest:

>Manually starts a transaction. You can run multiple statements within a transaction interactively, or non-interactively by batching commands and statements sequentially. Transactions that are not committed within 30 seconds will time out, and QLDB will reject any changes made during the transaction. For more details and examples, see the QLDB Developer Guide [link here?].